jest.config.ts

```diff
@@ -11,6 +11,9 @@ const jestConfig: JestConfigWithTsJest = {
     "default",
     ["jest-junit", { outputDirectory: "test-results/jest" }],
   ],
+  moduleNameMapper: {
+    "^(\\.{1,2}/.*)\\.js$": "$1",
+  },
```

Comment:
Borrowed this snippet from #170, as I have yet to clean up the Jest config and I don't want to spend too much time on it for this PR.